README.md

```diff
@@ -201,8 +201,8 @@ Also if an object is defined:
 case object Tick
 ```
 
-There will not be exceptions during serialization but Jackson will create
-another instance of `Tick` instead of restoring the `object Tick`'s underlying singleton.
+There will not be exceptions during serialization but, during deserialization, Jackson will create
+another `Tick` object instead of restoring the `object Tick`'s underlying singleton.
```

Comment:
Huh this is still pretty confusing... `object` in the sense of Scala (singleton), or in the sense of JVM (instance of a class)?

Reply:
I'd say:
```suggestion
another instance of `object Tick`'s underlying class instead of restoring the `object Tick`'s underlying singleton.
```

Reply:
Ok, done as suggested